docs/fud/xilinx.md

```diff
@@ -91,6 +90,19 @@ You will also need to configure the stage to point to your installations of [Vit
     xilinx_location: /scratch/opt/Xilinx/Vitis/2020.2
     xrt_location: /opt/xilinx/xrt
 
+The setup scripts can also be invoked manually.
+
+    source <Vitis_install_path>/Vitis/2020.1/settings64.sh
+    source /opt/xilinx/xrt/setup.sh
+
+On some Ubuntu distributions, you must also export LIBRARY\_PATH.
+
+    export LIBRARY_PATH=/usr/lib/x86_64-linux-gnu
+
+To execute a Calyx program on an FPGA, add a `toplevel` annotation to the main component to configure the top-level function of the RTL design. 
+
+    component main<"toplevel"=1>() -> ()
```

Comment:
Maybe I'm missing something, but I haven't noticed needing to do this—for example, [the example I've been using in these docs](https://github.com/cucapra/calyx/blob/master/examples/tutorial/language-tutorial-iterate.futil) seems to work OK, and its `main` doesn't have an annotation. Is there something you know here that I don't?

Reply:
True. I also ran the example without annotation and it seems to work fine. I guess there was a previous commit that resolved this issue, which I may have forgotten. I had this comment added in the beginning of this debugging session since examples could not even be passed onto the `fud` workflow if the annotation was not there. I'll try to look again.

Reply:
OK, cool. Sounds great. I think this is the operative advice now:
https://github.com/cucapra/calyx/blob/1e909eea93b7c8e3e3305d9d25132587c1e2fdd8/calyx/src/ir/from_ast.rs#L154

i.e., you can either call it `main` or annotate it with `toplevel`. But I don't think that's Xilinx- or synthesis-specific; it looks like *all* Calyx programs require this. (So maybe we do not need to mention it here.)

Reply:
> OK, cool. Sounds great. I think this is the operative advice now:
> 
> https://github.com/cucapra/calyx/blob/1e909eea93b7c8e3e3305d9d25132587c1e2fdd8/calyx/src/ir/from_ast.rs#L154
> 
> i.e., you can either call it `main` or annotate it with `toplevel`. But I don't think that's Xilinx- or synthesis-specific; it looks like _all_ Calyx programs require this. (So maybe we do not need to mention it here.)

Noted.